datasets/samplemotordatalimerick.py

```diff
@@ -0,0 +1,692 @@
+###################################################################################################
+#
+# Copyright (C) 2024 Analog Devices, Inc. All Rights Reserved.
+# This software is proprietary to Analog Devices, Inc. and its licensors.
+#
+###################################################################################################
+"""
+Classes and functions for the Limerick Sample Motor Data Dataset
```

Comment:
Please add a link to the documentation here

Reply:
Documentation is currently on pr, link will be added when it is published.